controllers/v1/entity-types.js

```diff
@@ -22,4 +21,61 @@ module.exports = class EntityTypes extends Abstract {
     return "entityTypes";
   }
 
+   /**
+   * @api {post} /kendra/api/v1/entity-types/list
+   * Lists of entity types.
+   * @apiVersion 0.0.1
+   * @apiName Lists of entity types.
+   * @apiGroup Entity Types
+   * @apiHeader {String} X-authenticated-user-token Authenticity token
+   * @apiParamExample {json} Request-Body:
+   * {
+    "query" : {
+        "name" : "school"
+    },
+    "projection" : ["_id","name"]
+    }
+   * @apiSampleRequest /kendra/api/v1/entity-types/list
+   * @apiUse successBody
+   * @apiUse errorBody
+   * @apiParamExample {json} Response: 
+   * {
+   * "message": "Entity types fetched successfully",
+   * "status": 200,
+   * "result" : [
+   *  {
+   * "_id": "5d15a959e9185967a6d5e8a6",
+   *  "name": "school"
+   }]
+  }
+   */
+
+  /**
+   * Lists of entity types.
+   * @method
+   * @name find
+   * @param {Object} req - Requested data.
+   * @param {Object} req.body.query - Filtered data.
```

Comment:
@aman-tunerlabs - Why do we need so much projection and skip and search in entity types?